plugin.cmake

```diff
@@ -14,6 +14,11 @@
 #  limitations under the License.
 ###############################################################################
 
+# support the "if(TEST ...)" command
+if(POLICY CMP0064)
```

Comment:
For cmake < 3.4, the `if(TEST)` statement will be ignored.  This means the #316 will still occur, but using the defaults will still work.